css/style.css

```diff
@@ -0,0 +1,224 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family: Helvetica;
+}
+.container {
+  width: 100%;
+  height: 100vh;
+  display: grid;
+  grid-template-columns: 10vw 1fr 10vw;
+  grid-template-rows: 100px 1fr 1fr 200px;
+}
+/* nav */
+.navbar {
+  grid-column: 1 / 4;
+  grid-row: 1 / 2;
+  background-color: lightgray;
+}
+.navbar ul {
+  grid-column: 2 / 3;
+  grid-row: 1 / 2;
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin-left: 120px;
+}
+.navbar li a {
+  margin: 0 1rem;
+  font-size: 1.5rem;
+  color: black;
+  text-decoration: none;
+}
+.logo {
+  grid-column: 2 / 3;
+  grid-row: 1 / 2;
+  margin-left: 120px;
+}
+
+.navbar li a:hover {
+  color: #bd3e3e;
+}
+/* navbar end */
+
+/* galleri start */
+.gallery {
+  display: grid;
+  height: 640px;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 100px 90px 25px 25px 100px 100px 80px;
+  grid-column: 2 / 3;
+  grid-row: 2 / 3;
+  margin: 5px 0;
+  gap: 15px;
+  padding: 15px;
+  background-color: #f5f5f5;
+}
+.gallery > div {
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  border-radius: 5px;
+  transition: all 0.5s;
+  opacity: 0.7;
+}
+.gallery > div:hover {
+  transform: scale(1.05);
+  opacity: 1;
+}
+.gallery > :nth-child(1) {
+  grid-column: 1 / 2;
+  grid-row: 1 / 2;
+  background-image: url("../images/1.jpg");
+}
+.gallery > :nth-child(2) {
+  grid-column: 2 / 3;
+  grid-row: 1 / 3;
+  background-image: url("../images/2.jpg");
+}
+.gallery > :nth-child(3) {
+  grid-column: 3 / 4;
+  grid-row: 1 / 5;
+  background-image: url("../images/3.jpg");
+}
+.gallery > :nth-child(4) {
+  grid-column: 4 / 5;
+  grid-row: 1 / 4;
+  background-image: url("../images/4.jpg");
+}
+.gallery > :nth-child(5) {
+  grid-column: 1 / 2;
+  grid-row: 2 / 6;
+  background-image: url("../images/5.jpg");
+}
+.gallery > :nth-child(6) {
+  grid-column: 2 / 3;
+  grid-row: 3 / 7;
+  background-image: url("../images/6.jpg");
+}
+.gallery > :nth-child(7) {
+  grid-column: 3 / 4;
+  grid-row: 5 / 8;
+  background-image: url("../images/7.jpg");
+}
+.gallery > :nth-child(8) {
+  grid-column: 4 / 5;
+  grid-row: 3 / 8;
+  background-image: url("../images/8.jpg");
+}
+.gallery > :nth-child(9) {
+  grid-column: 1 / 2;
+  grid-row: 6 / 8;
+  background-image: url("../images/9.jpg");
+}
+.gallery > :nth-child(10) {
+  grid-column: 2 / 3;
+  grid-row: 7 / 8;
+  background-image: url("../images/10.jpg");
+}
+
+/* gallery end */
+
+/* content start */
+.sec-container {
+  background: #f5f5f5;
+  grid-column: 2 / 3;
+  grid-row: 3 / 4;
+  height: 600px;
+  width: 100%;
+}
+.content {
+  display: grid;
+  height: 100%;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  gap: 5px;
+}
+
+.red {
```

Comment:
- Lorem text should have a text shadow. 
- To much repetition -> display flex , align center, justify center. Create one class with this properties and add to your section items.
eg. class="content-item bg-red"
DRY (don't repeat yourself)